src/app/containers/RadioPageBlocks/Blocks/AudioPlayer/index.jsx

```diff
@@ -66,9 +70,9 @@ const AudioPlayer = ({
 
     return (
       <OuterWrapper>
-        <InnerWrapper>
+        <MediaMessageWrapper>
```

Comment:
I don't think there's consistency within the codebase so this isn't really an issue, but I have seen `Styled` used as a key word within the component declaration to indicate that the component is a styled version of another component (in this case `InnerWrapper`). That being said `MediaMessageWrapper` is much more descriptive than `StyledInnerWrapper`. Just wanted to get your thoughts on best practice for when we do this?

Reply:
I think the `Styled` prefix is to indicate it's a component created by `styled-components`. It's maybe something we should discuss and then document though.